recipes/preseq/meta.yaml

```diff
@@ -5,7 +5,10 @@ package:
   version: {{ version }}
 
 build:
-  number: 5
+  number: 6
+  run_exports:
+    - {{ pin_subpackage("preseq", max_pin="x.x") }}
+  skip: True # [osx-arm64]
```

Comment:
IMO this skip is not needed. `osx-arm64` is not supported yet anyway.

Reply:
:man_shrugging: if you see the logs of previous commit ([ab85952](https://github.com/bioconda/bioconda-recipes/pull/46596/commits/ab85952ab39c299a395b7d636263819501d4c316)), it was failing for osx, so apparently it was not totally ignored.

Reply:
If the last build worked for osx-64, why did it fail in the last one if the only thing I’ve done is to add a new platform? The errors (in case there are any), should only appear for linux-aarch64 then, right?. If the conditional flag # [osx-arm64] is not affecting our arm64/aarch64 build, how has it solved the problem?. My feeling is that something in the CI pipeline is making the builds for osx-64 to be now executed by an osx-arm64 machine. Because the CI has evolved in between 9 months ago (when the recipe succeeded) and today, this might explain why it has started failing now. Do you think this reasoning is correct?. Any idea on how to continue?

Reply:
https://github.com/bioconda/bioconda-recipes/pull/46540#discussion_r1530362773 - here a Bioconda core team member explains that `osx-arm64` is not supported yet (this is a comment from yesterday).
Maybe the failure was a temporary glitch...
Please re-try by removing `  skip: True # [osx-arm64]`
If it fails again then try with `  skip: True # [osx]`